### PR TITLE
Add build version capability

### PR DESCRIPTION
**Pull request by AkioFaris**

### CODE REVIEW

**Comment by AkioFaris** [comment]
[14] - QvKspuIU

**Comment by AkioFaris** [comment]
[14] - Allure report on Netlify: https://5ecfca1665a4b6600d64bc77--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[15] - tXPgpDPd

**Comment by AkioFaris** [comment]
[15] - Allure report on Netlify: https://5ecfcbea65a4b6744364be0f--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[16] - 81STZVJ1

**Comment by AkioFaris** [comment]
[16] - Allure report on Netlify: https://5ecfcebfaede4cb4eb55f8a2--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[17] - HtB5GYXl

**Comment by AkioFaris** [comment]
[17] - Allure report on Netlify: https://5ecfd408a8e8adddcdb3f898--thirsty-yalow-da011e.netlify.app

File: pom.xml [hunk 1]
```diff
@@ -113,9 +113,4 @@
...
         </resources>
     </build>
-
-</project>
-
-        <!--
-        mvn clean package -Dreport.portal.user=test-user -Dreport.portal.password=Fqvq1s0S
-        -->
\ No newline at end of file
+</project>
```
Comment (mf-code) [inline pom.xml]: please add new line

**Review by mf-code — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (AkioFaris) [reply pom.xml]: Thanks, added

**Review by ViktarR — APPROVED** [review]

**Comment by AkioFaris** [comment]
[18] - W0e2VKQU

**Comment by AkioFaris** [comment]
[18] - Allure report on Netlify: https://5ecfd7b638bc5dbc33894277--thirsty-yalow-da011e.netlify.app

**Review by mf-code — APPROVED** [review]

**Pull request merged**